src/font.rs

```diff
@@ -353,6 +353,70 @@ impl Font {
             .guidelines
             .get_or_insert_with(Default::default)
     }
+
+    /// Returns a vector of immutable lib key string references
+    pub fn lib_keys(&self) -> Vec<&String> {
+        self.lib.keys().collect::<Vec<&String>>()
+    }
+
+    /// Returns an iterator over immutable lib key string references
+    pub fn iter_lib_keys(&self) -> impl Iterator<Item = &String> {
+        self.lib.keys()
```

Comment:
Instead of `&String`, I would prefer this return `&str`. This is not that important in practice, but is more idiomatic, and may let the caller avoid needing to explicitly convert to a `&str` on the calling side.
```suggestion
    pub fn iter_lib_keys(&self) -> impl Iterator<Item = &str> {
        self.lib.keys().map(String::as_ref)
```

I'd also just call this `lib_keys`.